src/main/java/io/anserini/kg/freebase/FreebaseNode.java

```diff
@@ -125,4 +129,30 @@ public void clear() {
     subject = null;
     predicateValues = null;
   }
+
+  public static String cleanUri(String uri) {
+    if (uri.charAt(0) == '<')
+      return uri.substring(1, uri.length() - 1).toLowerCase();
+    else
+      return uri;
+  }
+
+  public static RdfObjectType getObjectType(String objectValue) {
+    // Determine the type of this N-Triples 'value'.
+    switch (objectValue.charAt(0)) {
+      case '<':
+        // e.g., <http://rdf.freebase.com/ns/m.02mjmr>
+        return RdfObjectType.URI;
+      case '"':
+        if (objectValue.charAt(objectValue.length() - 1) == '"') {
+          // e.g., "Hanna Bieluszko"@en";
```

Comment:
is this RdfObjectType.STRING or RdfObjectType.TEXT? 
the test shows contradictory examples.